pkg/utils/requester.go

```diff
@@ -100,40 +98,40 @@ func NewHTTPRequester(url string, params ...func(*HTTPRequester)) *HTTPRequester
 }
 
 // Get executes HTTP GET with url and optional extra headers, returns body in []bytes
-// url created as url+sdkKey.json
-func (r HTTPRequester) Get(headers ...Header) (response []byte, responseHeaders http.Header, code int, err error) {
-	return r.Do("GET", nil, headers)
+// url created as api+sdkKey.json
```

Comment:
This comment about `api+sdkKey.json` is very specific to the datafile. The requester shouldn't care about this detail. We can just remove this part of the comment.